install/linux/usr/share/odemis/sim/meteor-fibsem-sim.odm.yaml

```diff
@@ -84,7 +90,7 @@ METEOR-FIBSEM-Sim: {
                 "Sample pre-tilt":  0.6108652381980153},  # 35°
         FAV_FM_POS_ACTIVE: {"rx": 0.12213888553625313  , "rz":  3.141592653589793}, # 7° - 270°
         FAV_SEM_POS_ACTIVE: {"rx": 0.6108652381980153, "rz": 0},  # pre-tilt 35°
-        FAV_MILL_POS_ACTIVE: {"rx": 0.314159, "rz": 0},    # Note that milling angle (rx) can be changed per session  
+        # FAV_MILL_POS_ACTIVE: {"rx": 0.314159, "rz": 0},    # Note that milling angle (rx) can be changed per session  
```

Comment:
why is this commented?

Reply:
The metadata item isn't added until another PR: https://github.com/delmic/odemis/pull/3005/files#diff-a2c9bc5144464d40c3b2bb45d972506662f62a97029ccb1dac0f0ac3ea68274f. It'll be uncommented then